src/backend/vulkan/src/lib.rs

```diff
@@ -1081,6 +1098,9 @@ impl adapter::PhysicalDevice<Backend> for PhysicalDevice {
         if features.inherited_queries != 0 {
             bits |= Features::INHERITED_QUERIES;
         }
+        if self.supports_extension(*MESH_SHADER) {
+            bits |= Features::MESH_SHADER;
```

Comment:
what about `TASK_SHADER` bit?

Reply:
Added. Can't be separated without VkPhysicalFeatures2. Most probably inconsequential as DirectX 12 requires both shaders to be supported despite Task being optional in the pipeline as well as in the Vulkan extension itself.